ISSUE_TEMPLATE/enhancement.yaml

```diff
@@ -0,0 +1,23 @@
+name: Enhancement Tracking Issue
+description: Provide supporting details for a feature in development
+labels:
+  - enhancement
+body:
+  - type: textarea
+    id: background
+    attributes:
+      label: Background
+      description: |
+        Before explain the feature, please provide some background and lacking points.
+        What is lacking? What is needed? Why is it needed?
+    validations:
+      required: true
+
+  - type: textarea
+    id: todos
+    attributes:
+      label: TODOs
+      value: |
+        ```
+        - [ ] TODO: Write your TODOs here
+        ```
```

Comment:
leave an indent for the last line of any file

Reply:
Handled in [b3b2ca6](https://github.com/ajktown/docs/pull/77/commits/b3b2ca6b34e456e68c86804ef1bbc7802a738963)

Reply:
Handled in [56b9148](https://github.com/ajktown/docs/pull/77/commits/56b9148f14feaf5b4faef3746fd82a6c6c11ff73)